src/main.ts

```diff
@@ -1,16 +1,25 @@
-import App from './app/App.vue'
-import { TOKENS, get } from '@/services'
+import { TOKENS as $, services as production } from '@/services/production'
+import { build } from '@/services/utils'
 
 async function mountVueApplication() {
-  const env = get(TOKENS.env)
-  const kumaApi = get(TOKENS.api)
-  // During development setBaseUrl also optionally installs MSW mocking via MockKumaApi
-  kumaApi.setBaseUrl(env('KUMA_API_URL'))
-
-  const app = await get(TOKENS.app)(App)
+  const get = build(
```

Comment:
`get` is now returned by `build` and should (in the future) be the only place we should need to use it i.e. start up the application straight after we've built the service container.